rosa_getting_started/rosa-getting-started.adoc

```diff
@@ -29,17 +29,17 @@ You can create a ROSA cluster either with or without the AWS Security Token Serv
 include::modules/rosa-getting-started-environment-setup.adoc[leveloffset=+1]
 include::modules/rosa-getting-started-enable-rosa.adoc[leveloffset=+2]
 include::modules/rosa-getting-started-install-configure-cli-tools.adoc[leveloffset=+2]
-include::modules/rosa-getting-started-verify-elb-role.adoc[leveloffset=+2]
-include::modules/rosa-getting-started-verify-aws-quota.adoc[leveloffset=+2]
+//include::modules/rosa-getting-started-verify-elb-role.adoc[leveloffset=+2]
+//include::modules/rosa-getting-started-verify-aws-quota.adoc[leveloffset=+2]
 
 [id="rosa-getting-started-creating-a-cluster"]
 == Creating a ROSA cluster with STS
 
 Choose from one of the following methods to deploy a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS). In both scenarios, you can deploy your cluster by using {cluster-manager-first} or the ROSA CLI (`rosa`):
```

Comment:
Suggestion: If you say "In each scenario" vs. "In both scenarios", then you don't incur future inaccuracy should a new scenario become available.